### PR TITLE
Theme support wrapper

### PR DESCRIPTION
**Pull request by ankitrox**
Theme support wrapper class.
Useful as single point of reference for multiple locations like, Customizer, TinyMCE Module etc.

### CODE REVIEW

File: includes/ThemeSupport.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+ */
+
+namespace Google\Web_Stories;
+
+/**
+ * Class ThemeSupport.
+ *
+ * @package Google\Web_Stories
+ */
+class ThemeSupport {
```
Comment (spacedmonkey) [inline includes/ThemeSupport.php:31]: What is the point of this being a class? A class with static method. Why not just a function?

File: includes/ThemeSupport.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	 * Name for theme support.
+	 */
+	const NAME = 'web-stories';
+
+	/**
+	 * Get theme support and arguments for theme support.
+	 *
+	 * @return array
+	 */
+	public static function get_theme_support() {
```
Comment (spacedmonkey) [inline includes/ThemeSupport.php:43]: Where and how would this method be used?

**Comment by ankitrox** [comment]
@spacedmonkey I'm going to discard this PR. Calling static method from another class would create unnecessary coupling. Will go with simple function as of now, if things expands in "Theme Support", we can have separate object for it.

File: includes/ThemeSupport.php (hunk 2 above)
Comment (ankitrox) [reply includes/ThemeSupport.php:43]: This would be used in Customizer and Stories' wrapper function.

**Comment by ankitrox** [comment]
Closing this in favour of #182

**Pull request closed without merging**